content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1466,6 +1466,19 @@ This annotation records a comma-separated list of
 managed by [Node Feature Discovery](https://kubernetes-sigs.github.io/node-feature-discovery/) (NFD).
 NFD uses this for an internal mechanism. You should not edit this annotation yourself.
 
+### nfd.node.kubernetes.io/node-name
+
+Type: Annotation
+
+Example: `nfd.node.kubernetes.io/node-name: node-1`
+
+Used on: Nodes
+
+It specifies which node the NodeFeature object is targeting.
+Creators of NodeFeature objects must set this label and 
+consumers of the objects are supposed to use the label for 
+filtering features designated for a certain node.
+
 {{< note >}}
 These annotations only applies to nodes where NFD is running.
```

Comment:
```suggestion
This annotation only applies to nodes where NFD is running.
```
Would you mind to change this grammar issue?

Reply:
@windsonsea 
Note is written for [nfd.node.kubernetes.io/**] annotations.  
how about below content

> Node Feature Discovery(NFD) annotations only applies to the nodes where NFD is running.

Reply:
Main concern is that `annotations` does not match `applies`.
If we add NFD, it may look like:
```
These Node Feature Discovery (NFD) annotations only apply to the nodes where NFD is running.
```

Reply:
Understood, i will make the proposed changes.

Reply:
As far as i know
1. it is a label
2. [Note] for above 5 nfd.node.kubernetes.io/*
3. These annotation are often used  for NFD
Example:
https://github.com/kubernetes-sigs/node-feature-discovery/blob/02dd550d3bacbae6b24ded0079d00caba09b5c4b/docs/usage/customization-guide.md?plain=1#L58